packages/office-ui-fabric-react/.storybook/decorators/withKeytipLayer.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import { KeytipLayer } from 'office-ui-fabric-react';
+
+export const withKeytipLayer = storyFn => {
```

Comment:
Should we put this in a shared location so other packages can use it too? Or do we only care about the current version of keytips working in OUFR?

Reply:
maybe, but current use cases do not require this to be duplicated (hence, to be shared). 
We might need a new package to put shared decorators (I don't think `/scripts` is the right place). I'd to postpone this until we actually use this in multiple places